src/pages/index.tsx

```diff
@@ -1,5 +1,10 @@
-import { Homepage } from "~/containers/home/index.jsx";
+import { Homepage } from "~/containers/home/index.js";
+import { ReCaptchaProvider } from "~/containers/reCaptcha";
 
 export default function Home() {
-  return <Homepage />;
+  return (
+    <ReCaptchaProvider>
```

Comment:
Still don't understand why the context. Why not just a hook or component that is called inside Homepage for handling this?

Reply:
Yeah I know, as much as I like contexts, i tried to not use it however, It's to make sure you only have one instance of Google Recaptcha per page and it doesn't reload unecessarily when your components re-rendered.

Reply:
A token is generated and valid for 2 min, if it's constantly re-rendered, it cannot keep track if the same token is expired or not.

Reply:
Yes, I found a solution to not use a context, it required some hook prop drilling.